packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -1930,7 +1930,7 @@ function revisionFromTaggedChange(
 function revisionFromRevInfos(
 	revInfos: undefined | readonly RevisionInfo[],
 ): RevisionTag | undefined {
-	if (revInfos === undefined || revInfos.length !== 1) {
+	if (revInfos === undefined || revInfos.length !== 1 || revInfos[0] === undefined) {
```

Comment:
assert, don't change the logic. If, for whatever reason, `renInfos[0]` was the value `undefined`, the current behavior would throw an error. With this change, it will silently continue.